egs/multi_ja_en/ASR/local/utils/asr_datamodule.py

```diff
@@ -265,12 +295,16 @@ def train_dataloaders(
             logging.info("Loading sampler state dict")
             train_sampler.load_state_dict(sampler_state_dict)
 
+        seed = torch.randint(0, 100000, ()).item()
```

Comment:
Is this seed a fixed value? If not, I think it is better to fix seeds before using random functions.